README.md

```diff
@@ -343,7 +343,7 @@ In order to perform local development on this codebase:
 
 * Check out the repository: `git clone https://github.com/hasura/ndc-typescript-deno.git`
 * This assumes that you will be testing against function in `./functions`
-* Vendor the dependencies with `cd ./function && deno vendor -f index.ts`
+* Vendor the dependencies with `cd ./function && deno vendor --node-modules-dir -f index.ts`
 * Serve your functions with `deno run -A --watch --check ./src/mod.ts serve --configuration <(echo '{"functions": "./functions/index.ts", "vendor": "./functions/vendor", "schemaMode": "INFER"}')`
```

Comment:
Likewise this is the default functions location.

Reply:
Not sure what this comment is referring to?

Reply:
Defaulting around prevendor behaviour.